README.md

```diff
@@ -25,7 +25,39 @@ The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool
     <img src="docs/images/output/overview/eager2_workflow.png" alt="nf-core/eager schematic workflow" width="70%"
 </p>
 
-## Pipeline steps
```

Comment:
Did this intentionally creep in?

Reply:
No, did some reshuffling to match the nf-core template a bit :+1: